homeassistant/components/fritz/common.py

```diff
@@ -670,6 +670,10 @@ async def async_get_wan_link_properties(self) -> dict[str, Any]:
             partial(self.get_wan_link_properties)
         )
 
+    async def async_ipv6_active(self) -> bool:
+        """Call WANIPConn service."""
```

Comment:
I'm not really happy with this description ... maybe something like:
```suggestion
        """Call WANIPConn service to check whether an ipv6 is active."""
```